common/src/main/java/com/wynntils/core/persisted/upfixers/UpfixerManager.java

```diff
@@ -58,6 +59,7 @@ public UpfixerManager() {
         registerConfigUpfixer(new NpcDialoguesOverlayConfigsMovedUpfixer());
         registerConfigUpfixer(new TowerAuraVignetteNameUpfixer());
         registerConfigUpfixer(new TowerAuraVignetteAndOverlayMovedToCommonFeature());
+        registerConfigUpfixer(new TradeMarketAutoOpenChatToTradeMarketQuickSearchUpfixer());
```

Comment:
warning! This is not alphabetical ordering! These upfixers are run in order, so you should move this down to be the last one.

Reply:
[ec634d5](https://github.com/Wynntils/Wynntils/pull/2908/commits/ec634d5e280ae9ca0ccffe7119e63de4ce586476)